diff --git a/src/train.py b/src/train.py
index b28587f..7683b07 100644
--- a/src/train.py
+++ b/src/train.py
@@ -4,6 +4,7 @@ import sys
 from argparse import ArgumentParser
 
 import torch
+import wandb
 from torch.utils.tensorboard import SummaryWriter
 
 import dataloader
@@ -22,6 +23,7 @@ def main():
     logging.getLogger().setLevel(logging.INFO)
 
     # Tensorboard Logs
+    wandb.init(sync_tensorboard=True)
     suffix = 0
     while os.path.exists(f"{os.path.dirname(os.path.abspath(__file__))}/logs/{config.exp_name}_{suffix}"):
         suffix += 1
